client/extensions/woocommerce/index.js

```diff
@@ -12,6 +12,7 @@ import { navigation, siteSelection } from 'my-sites/controller';
 import { renderWithReduxStore } from 'lib/react-helpers';
 import ProductCreate from './app/products/product-create';
 import Dashboard from './app/dashboard';
+import Checkout from './app/settings/checkout';
```

Comment:
To disambiguate from a potential future Checkout component, how about SettingsCheckout or CheckoutSettings?